db/controllers/allArticlesController.js

```diff
@@ -0,0 +1,8 @@
+
+const { selectAllOfArticles } = require('../models/allArticlesModel');
+
+exports.getAllArticles = (req, res, next) => {
+  selectAllOfArticles().then((data) => {
+    res.status(200).send(data);
+  })
```

Comment:
The common convention when serving data is not just to respond with an array of the data, but to wrap it in an object and put it on a descriptive property (like in this case, `articles`)